### PR TITLE
HTTPS everywhere, fixes some loading errors

### PR DESCRIPTION
**Pull request by abitrolly**
Linefeed at the end is added by `vim`. Used this recipe for search and replace.

https://vi.stackexchange.com/questions/2776/vim-search-replace-all-files-in-current-project-folder/10310#10310

Currently https://openstreetmap.by/ is not loading the map, because of `http://` links are blocked.

image

### CODE REVIEW

**Comment by abitrolly** [comment]
@Komzpa @tbicr ping,

File: static/opensearch.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
```
Comment (tbicr) [inline static/opensearch.xml:2]: is HTTP -> HTTPS correct change for namespace?

**Review by tbicr — COMMENTED** [review]
Current changes looks good with a few comments:

1. lets check is it the right way to change namespace in openseach.xml
2. this PR does not fix real issue of map loading, as it happens on `http://openstreetmap.by/data/v3.json` load that configured there https://github.com/osm-by/OpenStreetMap.by/blob/master/.env#L11

**Comment by tbicr** [comment]
added fix allows flexible schema and domain: https://github.com/osm-by/OpenStreetMap.by/pull/34

File: static/opensearch.xml (hunk 1 above)
Comment (abitrolly) [reply static/opensearch.xml:2]: Well, I forgot that XML namespaces are not URLs. It is a mistake of course. Why they put `http://` there is the first place? It is plain confusing.

**Comment by abitrolly** [comment]
You could just add the stuff on top of this PR. It allows modifications.

**Review by tbicr — APPROVED** [review]

**Comment by abitrolly** [comment]
I am not able to merge changes in this repo.

**Pull request merged**